### DISCUSSION TITLE
Important Notes for Gssoc24-Ext Mentors

### DISCUSSION THREAD

**Discussion by AlfiyaSiddique** [AlfiyaSiddique/TastyTrails#21] [Announcements]
Hello to all the GSSOC24-Ext mentors!
Welcome to TastyTrails!🎉
Greetings from Project Admin -Alfiya :)

As you will be managing the repo and all the issues and PR please follow the below instructions:
1. Assign issues on a first come first serve basis.
2. Do not accept issues with general descriptions, ask for details, like:
   a) What the contributor will change in the web app?
   b) How it will impact the end users.
   c) What is the problem with the existing feature or design
3. While Assigning make sure the issue is not duplicated.
4. Merge PRs only after my review.
5. Do not merge PRs if the contributor has not opened an issue for it. Ask them to open the issue if the PR looks worthy else close it off

**Violating these rules or being inactive can result in the cancellation of your mentorship from this repo.**

You can also help us by adding some guidelines like steps to setting up projects. steps to open issues etc.
Let's make this project the best in this GSSOC24-Ext event. Thanks for your support.

**Comment by its-pratyushpandey** [comment]
Thank you, Alfiya! Excited to be part of TastyTrails and support GSSOC25-Ext. I’ll follow all the guidelines strictly, ensure proper issue/PR management, and help maintain project quality. Looking forward to a great collaboration! 🚀

**Reply by AlfiyaSiddique** [reply]
Thank you so much!  Just a small clarification — TastyTrails was part of GSSOC 2024 and not GSSOC 2025-Ext.  But feel free to explore the repo, collaborate, and contribute wherever you can.

**Reply by its-pratyushpandey** [reply]
> Thank you so much! Just a small clarification — TastyTrails was part of GSSOC 2024 and not GSSOC 2025-Ext. But feel free to explore the repo, collaborate, and contribute wherever you can.

 Alfiya. I appreciate the opportunity to be a part of TastyTrails and will ensure that all contributions align with the project’s objectives. I look forward to collaborating effectively and adding meaningful value to the repository. By the way you are so consistant on github , thats why i appreciated it with stars

**Reply by AlfiyaSiddique** [reply]
Thank you! I appreciate your initiative.
